src/plugins/plot/MctPlot.vue

```diff
@@ -795,7 +794,7 @@ export default {
       };
       this.config.xAxis.set('range', newRange);
       if (!isTick) {
-        this.annotatedPoints = [];
+        this.annotatedPointsBySeries = [];
```

Comment:
Hmm. Above we're assigned this to an object `{}`. Here it's an array. Can you change the erroneous ones?